src/components/Reviews.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class Reviews extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  
+  render() {
+    return(
+      <div id="reviews">
+        <h3>Ratings Component</h3>
```

Comment:
I think you mean Reviews Component

Reply:
Should have made more tests lol. Good thing it's just a placeholder. Thanks!